packages/react-slider/src/components/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,345 @@
+import * as React from 'react';
+import { safeCreate, safeMount } from '@fluentui/test-utilities';
+import { resetIds, KeyCodes } from '@fluentui/utilities';
+import { Slider } from './Slider';
+import { isConformant } from '../../common/isConformant';
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+describe('Slider', () => {
+  beforeEach(() => {
+    resetIds();
+  });
+
+  it('renders Slider correctly', () => {
+    safeCreate(<Slider defaultValue={5} />, component => {
+      const tree = component.toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+  });
+
+  isConformant({
+    Component: Slider,
+    displayName: 'Slider',
+    disabledTests: ['kebab-aria-attributes'],
+  });
```

Comment:
You should probably move this to before the first test, right after `beforeEach`.